src/core/vdom.ts

```diff
@@ -1312,9 +1337,13 @@ export function renderer(renderer: () => RenderResult): Renderer {
 			const isEvent = propName.substr(0, 2) === 'on' || onlyEvents;
 			const eventName = onlyEvents ? propName : propName.substr(2);
 			if (isEvent && !properties[propName]) {
-				const eventCallback = _eventMap.get(previousProperties[propName]);
-				if (eventCallback) {
-					domNode.removeEventListener(eventName, eventCallback);
+				const proxyEvents = _eventMap.get(domNode) || {};
+				let proxyEvent: { proxy: EventListener; callback: Function; options: any } | undefined =
```

Comment:
Could this be `EventMapValue | undefined`?

Reply:
yeah it can actually just be inferred, i just forgot to remove it.